src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int firstSide;
+
+    public Square(int firstSide, Color color) {
+        this.firstSide = firstSide;
+        super.color = color;

```

Comment:
It's not recommended to assign values to the superclass fields outside of the constructor. Instead, you should pass the color to the superclass constructor using `super(color);`.